apps/api-extractor/src/DebugRun.ts

```diff
@@ -26,6 +26,10 @@ const extractor: Extractor = new Extractor(
   }
 );
 
-extractor.analyzeProject();
+if (!extractor.processProject()) {
+  console.log('processProject() failed the build');
```

Comment:
console.error

Reply:
This file is only used for easier debugging during development; it's not part of the shipping code

---
In reply to: [152713831](https://github.com/Microsoft/web-build-tools/pull/451#discussion_r152713831) [](ancestors = 152713831)